platform/features/timeline/bundle.js

```diff
@@ -127,7 +127,16 @@ define([
                             14400000,
                             28800000,
                             43200000,
-                            86400000
+                            86400000,
+                            86400000 * 2,
+                            86400000 * 5,
+                            86400000 * 10,
+                            86400000 * 20,
+                            86400000 * 30,
+                            86400000 * 60,
+                            86400000 * 120,
+                            86400000 * 240,
+                            86400000 * 365
```

Comment:
Fair enough, I figured that was the case, but thought that since in most cases the value is approx. doubled there might be an easier way. If this gives a better user experience, then leave it as is. Curse those Sumerians and their base 60 number system.